README.md

```diff
@@ -139,6 +146,13 @@ const MyPage = observer(() => {
     props => new MyPageViewModel(props, userStore), // Factory-function will be called only once on first render
     { userId } // These `props` are reactive. It will be passed and updated in view-model every time it changes without creating new instance of view-model 
   );
+  
+  //** or
+  // const viewModel = useViewModel(
+  //    MyPageViewModel, 
+  //   { userId },
+  //   [userStore] // Extra dependencies for the view-model constructor
```

Comment:
I think we should add this new param to `API documentation` as `useViewModel(ViewModelClass, props?, args?)`.
Also, it would be great to add an example of using it to storybook docs.